src/status_im/contexts/wallet/collectible/tabs/overview/view.cljs

```diff
@@ -64,7 +62,12 @@
         :subtitle      subtitle}]]]))
 
 (defn view
-  []
-  [:<>
-   [info]
-   [traits-section]])
+  [collectible]
+  (let [owner-account (rf/sub [:wallet/collectible-details-owner collectible])
+        traits        (-> collectible :collectible-data :traits)]
+    [:<>
+     [info
+      {:chain-id (-> collectible :id :contract-id :chain-id)
+       :account  owner-account}]
+     (when (pos? (count traits))
+       [traits-section traits])]))
```

Comment:
Made this component more flexible and reusable, previously we had subscriptions that only worked as extractors, now when we want to extract a value we just do it from the `collectible` received.